azure-iot-hub/samples/iothub_job_manager_method_sample.py

```diff
@@ -0,0 +1,67 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import sys
+import os
+import uuid
+import time
+import datetime
+import msrest
+import pprint
+from azure.iot.hub import IoTHubJobManager
+from azure.iot.hub.models import JobProperties, JobRequest, CloudToDeviceMethod
+
+
+iothub_connection_str = os.getenv("IOTHUB_CONNECTION_STRING")
+
+
+def create_method_job_request():
+    job = JobRequest()
+    job.job_id = "methodJob-" + str(uuid.uuid4())
+    job.type = "scheduleDeviceMethod"
+    job.cloud_to_device_method = CloudToDeviceMethod(
+        method_name="test_method",
+        payload={"key": "value"},
+        response_timeout_in_seconds=30,
+        connect_timeout_in_seconds=15,
+    )
```

Comment:
import/export and method call used to be a single sample.  I separated them for clarity.  These 2 fields were the only significant change needed to make the sample actually run.